app/locale/he.coffee

```diff
@@ -237,61 +237,61 @@ module.exports = nativeDescription: "עברית", englishDescription: "Hebrew",
-#    reload_confirm: "Reload All"
+    reload_title: "?לטעון מחדש את כל הקוד"
+    reload_really: "?אתה בטוח שאתה רוצה לטעון שוב את השלב הזה ולהתחיל מההתחלה"
+    reload_confirm: "טען הכול שוב"
     victory: "ניצחון"
     victory_title_prefix: ""
-    victory_title_suffix: " Complete"
-#    victory_sign_up: "Sign Up to Save Progress"
-#    victory_sign_up_poke: "Want to save your code? Create a free account!"
-#    victory_rate_the_level: "Rate the level: " # Only in old-style levels.
-#    victory_return_to_ladder: "Return to Ladder"
-#    victory_play_continue: "Continue"
-#    victory_saving_progress: "Saving Progress"
-#    victory_go_home: "Go Home" # Only in old-style levels.
-#    victory_review: "Tell us more!" # Only in old-style levels.
-#    victory_hour_of_code_done: "Are You Done?"
-#    victory_hour_of_code_done_yes: "Yes, I'm finished with my Hour of Code™!"
-#    victory_experience_gained: "XP Gained"
-#    victory_gems_gained: "Gems Gained"
-#    guide_title: "Guide"
-#    tome_minion_spells: "Your Minions' Spells" # Only in old-style levels.
-#    tome_read_only_spells: "Read-Only Spells" # Only in old-style levels.
-#    tome_other_units: "Other Units" # Only in old-style levels.
-#    tome_cast_button_run: "Run"
-#    tome_cast_button_running: "Running"
-#    tome_cast_button_ran: "Ran"
-#    tome_submit_button: "Submit"
-#    tome_reload_method: "Reload original code for this method" # Title text for individual method reload button.
-#    tome_select_method: "Select a Method"
-#    tome_see_all_methods: "See all methods you can edit" # Title text for method list selector (shown when there are multiple programmable methdos).
-#    tome_select_a_thang: "Select Someone for "
-#    tome_available_spells: "Available Spells"
-#    tome_your_skills: "Your Skills"
-#    tome_help: "Help"
-#    tome_current_method: "Current Method"
-#    hud_continue_short: "Continue"
-#    code_saved: "Code Saved"
-#    skip_tutorial: "Skip (esc)"
-#    keyboard_shortcuts: "Key Shortcuts"
-#    loading_ready: "Ready!"
-#    loading_start: "Start Level"
-#    problem_alert_title: "Fix Your Code"
-#    problem_alert_help: "Help"
-#    time_current: "Now:"
-#    time_total: "Max:"
-#    time_goto: "Go to:"
-#    infinite_loop_try_again: "Try Again"
-#    infinite_loop_reset_level: "Reset Level"
+    victory_title_suffix: " הושלם"
+    victory_sign_up: "הירשם על מנת לשמור התקדמות"
+    victory_sign_up_poke: "!רוצה לשמור את הקוד? הירשם בחינם עכשיו"
+    victory_rate_the_level: " :דרג את השלב" # Only in old-style levels.
+    victory_return_to_ladder: "חזור לסולם"
+    victory_play_continue: "המשך"
+    victory_saving_progress: "שומר התקדמות"
+    victory_go_home: "חזור הביתה" # Only in old-style levels.
+    victory_review: "!ספר לנו עוד" # Only in old-style levels.
+    victory_hour_of_code_done: "?סיימת"
+    victory_hour_of_code_done_yes: "Yes, I'm finished with my Hour of Code™!" #I am not sure how to translate it with the [TM] sign
```

Comment:
Ok I will write it in the next update.
P.S: Just wondering, how much time does it take until the merged branch is updated on the site itself?